posthog/api/test/test_feature_flag.py

```diff
@@ -960,7 +965,7 @@ def test_getting_flags_is_not_nplus1(self) -> None:
                 format="json",
             ).json()
 
-        with self.assertNumQueries(9):
+        with self.assertNumQueries(15):
```

Comment:
same, n+1, not good, should be 11

Reply:
whatever you think is best 😂 - I said 11 because the one above was 11, but paid no attention